src/Microsoft.AspNetCore.Hosting/Startup/StartupExceptionPage.cs

```diff
@@ -243,13 +242,28 @@ private static string GenerateFooterEncoded()
 #else
             var clrVersion = HtmlEncodeAndReplaceLineBreaks(Environment.Version.ToString());
 #endif
-            var runtimeArch = HtmlEncodeAndReplaceLineBreaks(environment.RuntimeArchitecture);
+            var runtimeArch = HtmlEncodeAndReplaceLineBreaks(RuntimeInformation.ProcessArchitecture.ToString());
             var currentAssembly = typeof(StartupExceptionPage).GetTypeInfo().Assembly;
             var currentAssemblyVersion = currentAssembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
             currentAssemblyVersion = HtmlEncodeAndReplaceLineBreaks(currentAssemblyVersion);
 
-            var os = HtmlEncodeAndReplaceLineBreaks(environment.OperatingSystem);
-            var osVersion = HtmlEncodeAndReplaceLineBreaks(environment.OperatingSystemVersion);
+            var osName = string.Empty;
+            var osVersion = string.Empty;
```

Comment:
Couldn't we just use the `OSDescription` field on `RuntimeInformation`? I think it includes the version